ui/app/components/pending-tx.js

```diff
@@ -197,10 +197,10 @@ PendingTx.prototype.render = function () {
                 h(BNInput, {
                   name: 'Gas Price',
                   value: gasPriceBn,
-                  precision: 9,
-                  scale: 9,
-                  suffix: 'GWEI',
-                  min: MIN_GAS_PRICE_GWEI_BN.toString(10),
+                  precision: 6,
+                  scale: 6,
+                  suffix: 'MWEI',
```

Comment:
I disagree that we should be using MWEI units here--the general accepted units for gas manipulation is GWEI.

Reply:
You think it’s better to leave the min gas price at 1 gwei than convert to mwei?

Reply:
How much effort to add decimal support?

Reply:
I think a lot. I remember the amount of time spent on this one, to get it as stable as it is. Considering this directly affects funds spent, I don’t think it should be changed lightly.